### PR TITLE
[Phi] Add softmax with cross entropy infershape & yaml

### PR DESCRIPTION
**Pull request by chenwhql**
### PR types

Function optimization
### PR changes

OPs
### Describe

[Phi] Move softmax with cross entropy infershape & add yaml

softmax_with_cross_entropy原先的InferShape还不能移除，因为在npu和mlu的场景中参数不一样，多一个输出值，infershape逻辑也不同，因此新增对应infermeta用于新动态图接入

### CODE REVIEW

**Review by phlrain — APPROVED** [review]

**Review by XieYunshen — APPROVED** [review]
LGTM
增加单测执行时间
